src/component/recipe/RecipeFocusStart.jsx

```diff
@@ -54,40 +83,63 @@ const RecipeFocusStart = () => {
   //   getRecipeData();
   // });
 
+  const handleFavorite = () => {
+    // addToFavRecipes
+// if clicked set to false
+  if (!clicked) {
+    const newRecipeKey = firebase.database().ref().child(`users/${logStatus.uid}/favRecipes`).push().key;
+    const updates = {};
+    updates[`users/${logStatus.uid}/favRecipes/${newRecipeKey}`] = {id: newRecipeKey, backendId: faykeRecipeData.status.id}
+    firebase.database().ref().update(updates).catch((error) => console.error(error));
+    setClicked(true)
+  } else {
+    firebase.database().ref(`users/${logStatus.uid}/favRecipes/${newRecipeKey}`).remove().catch((error) => console.error(error));
+    setClicked(false)
```

Comment:
I don't think this was working...getting error about being able to access newRecipeKey (the key should probably be logStatus.favRecipes.recipeKey) where the recipeKey will be found by checking for the recipeId from the backend.